st2common/st2common/logging/formatters.py

```diff
@@ -73,7 +74,7 @@ def serialize_object(obj):
     if getattr(obj, 'to_dict', None):
         value = obj.to_dict()
     elif getattr(obj, 'to_serializable_dict', None):
-        value = obj.to_serializable_dict()
+        value = obj.to_serializable_dict(mask_secrets=True)
```

Comment:
I am fine with this. I don't see a way out either. People have to tell us what params are secrets. Sure, we can add in some auto-detection but that is down the lane.